docs/docs/bootstrapping.md

```diff
@@ -39,9 +39,9 @@ In this async mode, non-bootstrapped tables are replicated as normal by the main
 ***
 
 * a bootstrap starts with a document with `type = "bootstrap-start"`
```

Comment:
I dont think I completely understand. Do you want me to replace all the 4 bullet points with just single bullet point, saying `starts with  an event of type 'bootstrap-...'`?

I think being explicit is nice.